CMakeLists.txt

```diff
@@ -89,7 +89,7 @@ target_compile_features(${PROJECT_NAME} INTERFACE cxx_std_11)
 ################################################################################
 
 if(UREACT_TEST)
-    enable_testing()
+    include(CTest)
```

Comment:
It adds a lot of additional targets that is not convenient. Need to wrap it under a new option. Like 
```cmake
option(UREACT_СTEST "Generate the сtest targets for CI." n)
```

Reply:
Done